py/qqa/run.py

```diff
@@ -294,5 +295,13 @@ def write_tables(indir, outdir):
 
     nightsfile = os.path.join(outdir, 'nights.html')
     webpages.tables.write_nights_table(nightsfile, exposures)
+    try:
+        os.mkdir(os.path.join(outdir, 'cal_files'))
+    except OSError:
+        i = 0
+    files = ['bootstrap.js', 'bootstrap.css', 'bootstrap-year-calendar.css', 'bootstrap-year-calendar.js', 'jquery_min.js', 'popper_min.js']
+    for f in files:
+        copyfile(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'cal_files', f), os.path.join(outdir, 'cal_files', f))
+
```

Comment:
This works when running out of a git clone, but will likely break when we upgrade qqa to work with "python setup.py install".  We can future-proof this using pkg_resources, e.g. something like:
```python
from pkg_resources import resource_filename
for f in files:
    outfile = os.path.join(outdir, 'cal_files', f)
    if not os.path.exists(outfile):
        infile = resource_filename('qqa', os.path.join('cal_files', f))
        copyfile(infile, outfile)
```
Note: for that to work, I had to add a missing `qqa/__init__.py` file.  I pushed that to this branch already, but you'll need to `git pull` to get it.